build/tasks/generate-asar-task.coffee

```diff
@@ -0,0 +1,14 @@
+asar = require 'asar'
+path = require 'path'
+
+module.exports = (grunt) ->
+  {rm} = require('./task-helpers')(grunt)
+
+  grunt.registerTask 'generate-asar', 'Generate asar archive for the app', ->
+    done = @async()
+
+    appDir = grunt.config.get('atom.appDir')
```

Comment:
Check to make sure this dir exists, otherwise if you run `script/grunt generate-asar` explicitly, you regenerate an empty asar file